server_tests/test/clientTests.js

```diff
@@ -1185,7 +1225,8 @@ describe("RUST API TEST SUITE", function() {
             .then( function onSuccess (res) {
                 Comparison.oscTakePictureOutput(res);
                 Comparison.assertTrue(stopped);
-            }, wrapError);
+                deferred.resolve();
+            }, wrapError), deferred.promise]);
```

Comment:
This `then(deferred)` seems incorrect, it needs to be applied to the `bublPoll` promise I think?